specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -4299,40 +4315,50 @@
 		}],
 		"properties": {
 			"defaultLanguageCode": {
-				"type": "string",
-				"default": "en",
+				"$ref": "#/definitions/ImageAnalysisSkillSupportedLanguageCodes",
 				"description": "A value indicating which language code. Default is en."
 			},
 			"visualFeatures": {
 				"type": "array",
 				"items": {
-					"type": "string"
+					"$ref": "#/definitions/VisualFeatures",
+					"description": "The strings indicating what visual feature types to return."
```

Comment:
I don't know if you need a description inside of `items`. I think the description on the property should be enough.